k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -196,6 +197,22 @@ protected void onDetachedFromWindow() {
     }
 
     @Override
+    public void onFocusChanged(boolean hasFocus, int direction, Rect previous) {
+        super.onFocusChanged(hasFocus, direction, previous);
+        if (hasFocus) {
+            displayKeyboard();
+        }
+    }
+
+    private void displayKeyboard() {
+        InputMethodManager imm = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+        if (imm == null) {
+            return;
+        }
+        imm.showSoftInput(this, InputMethodManager.SHOW_IMPLICIT);
```

Comment:
`SHOW_IMPLICIT` is used when the app thinks the keyboard should be shown, as opposed to it was explicitly requested by the user and should be shown until explicitly closed again